### PR TITLE
Add new wrk for florencenet

### PR DESCRIPTION
**Pull request by akoptelov**

### CODE REVIEW

**Comment by akoptelov** [comment]
@adonagy Please review. Note that this uses slightly different version of the `tezos-node-bootstrap` image.

**Review by adonagy — APPROVED** [review]
LGTM, please squash those commits and merge into develop

File: .drone.yml [hunk 1]
```diff
@@ -5126,3 +5126,235 @@ steps:
...
+    - rm -f /home/tezos/data/lock
+    - cp $${BUILD_ARTIFACTS_PATH}/build_files/identities/identity_1.json /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1/identity.json
+    - tezos-node config reset --data-dir /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1 --network $${NETWORK} --no-bootstrap-peers
+    - tezos-node run --data-dir /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1 --history-mode archive --rpc-addr 0.0.0.0:8732 --net-addr 0.0.0.0:9734 --network $${NETWORK} --no-bootstrap-peers
+
+- name: wait-for-ocaml-snapshoted-nodes
+  image: simplestakingcom/tezos-node-bootstrap:latest
+  pull: if-not-exists
+  commands:
+    - tezos-node-bootstrap bootstrap --level=$${BLOCK_HEAD} --nodes $${OCAML_NODE_RPC_CONTEXT_ROOT_1} $${OCAML_NODE_RPC_CONTEXT_ROOT_2} $${OCAML_NODE_RPC_CONTEXT_ROOT_3}
```
Comment (bkontur) [inline .drone.yml:5197]: @akoptelov 
please, remove `${OCAML_NODE_RPC_CONTEXT_ROOT_2} $${OCAML_NODE_RPC_CONTEXT_ROOT_3}`  they are not used

Comment (akoptelov) [reply .drone.yml:5197]: Argh...

**Pull request merged**